Networking/Networking/Model/PaymentGateway.swift

```diff
@@ -0,0 +1,108 @@
+import Foundation
+
+/// Represents a Payment Gateway.
+///
+public struct PaymentGateway: Equatable {
+
+    /// Features for payment gateway.
+    ///
+    public enum Features: Equatable {
```

Comment:
nit: usually an enum name is singular? in the [Swift doc for enums](https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html) under "Enumeration Syntax", it says:

> Give enumeration types singular rather than plural names, so that they read as self-evident

Or this is a different concept in payment gateway?

```suggestion
    public enum Feature: Equatable {
```

Reply:
Totally! My mistake!